examples/01_tutorials/plot_first_level_model_details.py

```diff
@@ -107,41 +107,50 @@ def make_localizer_contrasts(design_matrix):
     contrasts = dict([(column, contrast_matrix[i])
                       for i, column in enumerate(design_matrix.columns)])
 
-    # Add more complex contrasts
-    contrasts['audio'] = (contrasts['clicDaudio']
-                          + contrasts['clicGaudio']
-                          + contrasts['calculaudio']
-                          + contrasts['phraseaudio']
-                          )
-    contrasts['video'] = (contrasts['clicDvideo']
-                          + contrasts['clicGvideo']
-                          + contrasts['calculvideo']
-                          + contrasts['phrasevideo']
-                          )
-    contrasts['computation'] = contrasts['calculaudio'] + contrasts['calculvideo']
-    contrasts['sentences'] = contrasts['phraseaudio'] + contrasts['phrasevideo']
-
+    contrasts["audio"] = (
```

Comment:
PEP8 recommends using single-quotes everywhere, except when enclosing single-quotes as part of the string.